ReactCommon/cxxreact/Instance.cpp

```diff
@@ -69,47 +69,39 @@ void Instance::initializeBridge(
 
 void Instance::loadApplication(std::unique_ptr<RAMBundleRegistry> bundleRegistry,
                                std::unique_ptr<const JSBigString> bundle,
-                               uint64_t bundleVersion, // TODO(OSS Candidate ISS#2710739)
-                               std::string bundleURL,
-                               std::string&& bytecodeFileName) { // TODO(OSS Candidate ISS#2710739)
+                               std::string bundleURL) { // TODO(OSS Candidate ISS#2710739)
```

Comment:
>// TODO(OSS Candidate ISS#2710739) [](start = 56, length = 34)

Remove?

Reply:
We do have a small change in naming, where upstream calls this sourcrUrl. I've seen some of the naming changes marked, so I think it makes sense to keep the comment since we still differ from upstream (admittedly not very much).

---
In reply to: [342826129](https://github.com/microsoft/react-native/pull/187#discussion_r342826129) [](ancestors = 342826129)